js/src/pages/price-benchmark/price/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import useAdsCurrency from '~/hooks/useAdsCurrency';
+import './index.scss';
+
+/**
+ * Price component to display a formatted price value with optional highlighting.
+ *
+ * @param {Object} props - The component props.
+ * @param {number} props.amount - The price value to be formatted and displayed.
+ * @param {boolean} [props.highlight=false] - Whether to apply a highlight style to the price.
+ * @return {JSX.Element} The rendered price component.
+ */
+const Price = ( { amount, highlight = false } ) => {
+	const { formatAmount } = useAdsCurrency();
+
+	return (
+		<span
+			className={ classnames( 'gla-price-benchmark-table__price', {
+				'gla-price-benchmark-table__price--highlight': highlight,
+			} ) }
+		>
+			{ formatAmount( amount ) }
```

Comment:
Should we validate that `amount` is a number and if a null/empty value is passed force it to a zero?

```js
const formattedAmount = amount ? formatAmount(amount) : formatAmount(0);
```